### PR TITLE
Add ThaiSum sentence segmentor

### PR DESCRIPTION
**Pull request by chameleonTK**
### What does this changes

Add a new sentence segmentor adopted from Chumpolsathien N., 2020. https://github.com/nakhunchumpolsathien/ThaiSum
It is a rule-based model from conjunction words and length of the sentences. 

### What was wrong

Just another option for the sentence segmentation model.

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/PyThaiNLP/pythainlp/blob/dev/CONTRIBUTING.md) to this repository.

- [/] Passed code styles and structures
- [/] Passed code linting checks and unit test

### CODE REVIEW

File: pythainlp/tokenize/thaisumcut.py [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+        text = text.replace("เวลาต่อมา", "เวลา<rth_toma>")
+        text = text.replace("อย่างไรก็ตาม", "อย่างไรก็ตาม")
+        text = text.replace("อย่างไรก็ตามหลังจาก", "<stop>อย่างไรก็ตาม<rth_langjak>")
+        text = text.replace("ซึ่งทำให้", "ซึ่ง<rth_tamhai>")
+        text = text.replace("โดยประมาท", "<doi>ประมาท")
+        text = text.replace("โดยธรรม", "<doi>ธรรม")
+        text = text.replace("โดยสัจจริง", "<doi>สัจจริง")
+
+        if "และ" in text:
+            tokens = pythainlp.word_tokenize(text.strip(), keep_whitespace=True)
```
Comment (wannaphong) [inline pythainlp/tokenize/thaisumcut.py]: Can you use replace `pythainlp.word_tokenize(text.strip(), keep_whitespace=True)` with `word_tokenize(text.strip(), keep_whitespace=True)` by `from pythainlp.tokenize import word_tokenize` at the head file?

File: pythainlp/tokenize/thaisumcut.py [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+@mastersthesis{chumpolsathien_2020, 
+    title={Using Knowledge Distillation from Keyword Extraction to Improve the Informativeness of Neural Cross-lingual Summarization},
+    author={Chumpolsathien, Nakhun}, 
+    year={2020}, 
+    school={Beijing Institute of Technology}
+"""
+
+import re
+import pandas as pd
+import pythainlp
```
Comment (wannaphong) [inline pythainlp/tokenize/thaisumcut.py]: Can you use replace `import pythainlp` with `from pythainlp.tokenize import word_tokenize`?

**Review by wannaphong — CHANGES_REQUESTED** [review]

**Comment by wannaphong** [comment]
I think you should change `import pythainlp.word_tokenize` to `from pythainlp.tokenize import word_tokenize` because `from pythainlp.tokenize import word_tokenize` is faster than `import pythainlp.word_tokenize` by specify the function.

**Comment by chameleonTK** [comment]
Fixed as suggested :D

**Comment by wannaphong** [comment]
@chameleonTK Can you fix pep8? If you can't fix, It is not problem because I can fix.

**Comment by chameleonTK** [comment]
> @chameleonTK Can you fix pep8? If you can't fix, It is not problem because I can fix.

I have fixed some but `line too long` warnings are not sensible to me so I kept them as they are. Feel free to fix them.

**Review by wannaphong — APPROVED** [review]
Thank you!

**Pull request merged**